ci/cloudbuild/cloudbuild.yaml

```diff
@@ -25,6 +25,7 @@ options:
     'COMMIT_SHA=${COMMIT_SHA}',
     'PR_NUMBER=${_PR_NUMBER}',
     'TRIGGER_TYPE=${_TRIGGER_TYPE}',
+    'GENERATE_GOLDEN_ONLY=""',
```

Comment:
Is this important, given that the above code doesn't distinguish between unset and empty?